pkg/types/powervs/powervs_regions.go

```diff
@@ -65,3 +69,15 @@ var Regions = map[string]Region{
 		Zones:       []string{"us-east"},
 	},
 }
+
+// VPCRegionForRegion returns the VPC region for the specified region.
+func VPCRegionForRegion(region string) (string, error) {
```

Comment:
I think if we rename the function to `VPCRegionForPowerVSRegion` it will be more clear